base/exports.jl

```diff
@@ -1010,7 +1010,6 @@ export
     Condition,
     consume,
     current_task,
-    istaskstarted,
```

Comment:
This removes an exported function without a deprecation. Is there now no way to tell that a task has started?